GitUI/UserControls/FileStatusDiffCalculatorInfo.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using GitUIPluginInterfaces;
+
+namespace GitUI
+{
+    public class FileStatusDiffCalculatorInfo
```

Comment:
😱 

This is only used in `FileStatusDiffCalculator`, why it is public? 
This should be a private struct nested within `FileStatusDiffCalculator`, placed in FileStatusDiffCalculator.FileStatusDiffCalculatorInfo.cs file.

All other .NET conventiontions must be followed as well, e.g. no non-private fields, camel case, etc.

Reply:
I really want to have this data class nested in FileStatusDiffCalculatorInfo.cs
Will change.